README.md

```diff
@@ -148,12 +148,23 @@ cp.Action = func() {
 }
 ```
 
-If you want you can add support for printing the app version (invoked by ```-v, --version```) like so:
+If you want, you can add support for printing the app version (invoked by ```-v, --version```) like so:
 
 ```go
 cp.Version("v version", "cp 1.2.3")
 ```
 
+If you want, you can add a long description for printing via ```--help``` like so:
```

Comment:
Not sure if `LongDesc` is needed in the `Basics` section.
Could you remove this part, since you do mention it again later in the README ?